README.md

```diff
@@ -14,7 +14,7 @@ To learn more about Pulp, [view the Pulp project page](https://pulpproject.org/)
 
 ## OpenAPI Spec
 
-View the latest version of the spec by [clicking here](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/ansible/galaxy_ng/master/openapi/openapi.yaml).
+View the latest version of the spec by [clicking here](https://console.redhat.com/api/automation-hub/v3/openapi.yaml).
```

Comment:
Not sure if we want to mention `CRC` in the upstream as a docs resource.

We have talked about replacing all the `automation-hub` references from upstream repo and use only `galaxy_ng`

So looks like we need to publish the open api spec somewhere else.

Ideally we would generate the openapi-spec as a static site and publish with the docs we have.